### PR TITLE
Adding bidali + Wording changes

### PR DESCRIPTION
**Pull request by dievazqu**

### CODE REVIEW

File: src/valora-dapp-list.json [hunk 1]
```diff
@@ -29,6 +35,14 @@
...
       "logoUrl": "https://raw.githubusercontent.com/valora-inc/app-list/main/assets/moola.png",
       "url": "https://app.moola.market/"
+    },
+    {
+      "name": "Bidali",
+      "id": "3",
+      "categoryId": "3",
+      "description": "dapps.bidali",
+      "logoUrl": "https://raw.githubusercontent.com/valora-inc/app-list/main/assets/bidali.png",
+      "url": "https://www.bidali.com/"
```
Comment (dievazqu) [inline src/valora-dapp-list.json:45]: I'm not sure about this though. Anyone can confirm?

Comment (gnardini) [reply src/valora-dapp-list.json:45]: Hmm, I think this one should go to the in-app screen, not be open in an external browser 🤔

**Review by gnardini — APPROVED** [review]
🚀

**Review by silasbw — APPROVED** [review]
:hot_pepper:

**Pull request merged**